Sources/MapboxNavigation/Feature.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+import Turf
+
+extension Feature {
+
+    func featureIdentifierToDouble() -> Double? {
```

Comment:
Identifier not be exposed as `Double`.

Reply:
I've run some tests for this part. The identifier in `Carbon` for filtering only accept the `Double` format, instead of the `Int64 ` in 1.x.